README.md

```diff
@@ -23,5 +23,9 @@ Roadmap
 
 Known Bug Workarounds
 =====================
-* Burrito launches off-center or on the wrong monitor
-  * This seems to be WM dependant. on Gnome it can be solved by holding down the super key and dragging burrito to the right position.
+* Burrito launches off-center or on the wrong monitor (This seems to be WM dependant.)
+  * On Gnome and KDE it can be solved by holding down the super key and dragging burrito to the right position.
+  * On KDE it is also possible to use window-rules to set the position or a target monitor automatically. \
+    If you are not yet familiar with window-rules, you may find the examples in the official documentation useful. [Link](https://docs.kde.org/stable5/en/kwin/kcontrol/windowspecific/examples.html) \
+    To put it simply, you can add a new rule if you go to `System Settings → Window Behavior → Window Rules`. \
+    Remember to set `window class (application)` to `burrito`.
```

Comment:
All the info looks like it is here. I think that this can be made more concise

```suggestion
  * On KDE it is also possible to use [window-rules](https://docs.kde.org/stable5/en/kwin/kcontrol/windowspecific/examples.html) to set the position or a target monitor automatically by adding a new rule to `System Settings → Window Behavior → Window Rules` and setting `window class (application)` to `burrito`.
```